mirrord/config/src/feature/split_queues.rs

```diff
@@ -0,0 +1,96 @@
+use std::collections::{BTreeMap, HashMap};
+
+use mirrord_analytics::{Analytics, CollectAnalytics};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::config::{ConfigContext, FromMirrordConfig, MirrordConfig};
+
+pub type QueueId = String;
+
+/// ```json
+/// {
+///   "feature": {
+///     "split_queues": {
+///       "first-queue": {
+///         "queue_type": "SQS",
+///         "message_filter": {
+///           "wows": "so wows",
+///           "coolz": "^very .*"
+///         }
+///       },
```

Comment:
```suggestion
///       "SQS/first-queue": {
///         "message_filter": {
///           "wows": "so wows",
///           "coolz": "^very .*"
///         }
///       },
```
I think following `target` pattern would make this more concise

Reply:
Now that I look at it again, I think it would be too much of a hassle.
`QueueId` would have to store queue type and then we'd have to manually check that the value variant matches. `untagged` sounds a bit scary, because we'd have to ensure that no two variants can be parsed from the same json.
tl;dr it's ok as it is